lib/balena.coffee

```diff
@@ -69,7 +67,7 @@ getSdk = (opts = {}) ->
 		logs: -> require('./logs')
 		settings: -> require('./settings')
 
-	defaults opts,
+	opts = Object.assign({
```

Comment:
This is technically different in that `opts = { apiUrl: undefined }` will no longer be defaulted, `opts = {}` however would. If you feel it's necessary I'm happy to emulate the previous behavior